.travis.yml

```diff
@@ -1,20 +1,40 @@
 language: java
-
-# Skip testing for now, since this is experimental
-script: ./gradlew build -x test
-
 jdk:
   - oraclejdk8
 
+# Anything after trusty doesn't have Java 8
 dist: trusty
 
 # Below skips the installation step completely (https://docs.travis-ci.com/user/customizing-the-build/#Skipping-the-Installation-Step)
-# If we don't skip it, Travis runs unnecessary Gradle tasks like './gradlew assemble'
-install:
-  - true
+# If we don't skip it Travis runs unnecessary Gradle tasks like './gradlew assemble'
+install: true
 
-# Only build on master and release tags
+# Build and test commits to master, PRs to master, and release tags
```

Comment:
The script is still doing only build (-x test), so this comment should be updated?

Reply:
Oops you're right. I blindly copied this from parseq. I'll update it